recursion/TowerOfHanoi.py

```diff
@@ -0,0 +1,14 @@
+def TowerOfHanoi(top, start, via, end):
```

Comment:
can you use the same format of placing the disks as in the existing code implemented in Java, also keep the same vairable names